### ISSUE TITLE
Graphics

### ISSUE THREAD

**Issue by VermiumSifell** [onlinesgc/Gamerbot-REST-API#4]
Är tanken att API:t ska skicka dessa till botten för att ladda upp till Discord?

**Comment by lukasabbe** [comment]
Nope, Grafiken används i generering av frames, som sedan sickas till Discord boten
